integration-tests/src/test/scala/chisel3/std/fifo/FifoSpec.scala

```diff
@@ -0,0 +1,117 @@
+// Author: Martin Schoeberl (martin@jopdesign.com)
+// License: this code is released into the public domain, see README.md and http://unlicense.org/
```

Comment:
As you are the author of this code and have to sign the CLA agreement to contribute to the repo, it seems we would want to keep the same license for ip-contributions as the rest of the repo. Thoughts?

Reply:
Yes, I am fine with the Chisel license. This was just the license used in ip-contribution. Will change it.